ERCS/ERC-xxxx

```diff
@@ -0,0 +1,174 @@
+---
+eip: xxxx
```

Comment:
```suggestion
eip: 7615
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.